docs/Diagnostics.md

```diff
@@ -0,0 +1,82 @@
+This document contains information about diagnostic tools and outputs used for debugging and diagnosing eBPF issues.
```

Comment:
Since we also have docs/debugging.md, it would help to disambiguate.

Reply:
Do you think it makes more sense to remove debugging.md and move its content into troubleshootingguide.md?

Reply:
Offhand, it may be too long so I'd suggest keeping them as separate documents, just treating Debugging.md as being a conceptual subsection of TroubleshootingGuide.md.

Reply:
I have added links into debugging.md from the troubleshooting guide.